dragonfly/implementations/roach_daq_run_interface.py

```diff
@@ -463,7 +463,7 @@ def trigger_settings(self):
         '''
         Returns all trigger settings
         '''
-        if self.trigger_type == None:
+        if self.acquisition_mode == None:
```

Comment:
shouldn't this check that acquisition_mode is 'triggering'?  if in streaming mode trigger settings won't exist because the fmt node doesn't exist?

Reply:
also do this same check at line 491?